homeassistant/components/dormakaba_dkey/strings.json

```diff
@@ -12,7 +12,7 @@
         "description": "[%key:component::bluetooth::config::step::bluetooth_confirm::description%]"
       },
       "reauth_confirm": {
-        "description": "The activation code is no longer valid, a new unused activation code is needed.\n\n"
+        "description": "The activation code is no longer valid, a new unused activation code is needed."
```

Comment:
As with all other violations, the line feeds seem intentional - but I am unsure why.

Reply:
It doesn't seem valid. If we want blank lines after the description in the UI that should be handled by the frontend, consistently.